cli/pcluster/config/mappings.py

```diff
@@ -925,6 +943,11 @@
         "validators": [url_validator],
         "update_policy": UpdatePolicy.IGNORED
     }),
+    ("cw_dashboard_template_url", {
```

Comment:
Why is this in the cluster section and not in the dashboard section?

Reply:
Good question, I introduced this parameter here to stay aligned with other custom_*_url parameters, but I think it can be a reasonable change.